prime-dotnet-webapi/Migrations/20220129015717_AddPharmacyTechnicianToa.cs

```diff
@@ -0,0 +1,679 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+using Prime.Models;
+
+namespace Prime.Migrations
+{
+    public partial class AddPharmacyTechnicianToa : Migration
+    {
+        private static readonly DateTimeOffset SEEDING_DATE = DateTimeOffset.Parse("2019-09-16 -7:00");
```

Comment:
why is this line in your migrations file? did you have to add this yourself?

Reply:
> why is this line in your migrations file? did you have to add this yourself?

yes, new AgreementVersion row needs the seeding date.

Reply:
re-gened